beater/connbeat.go

```diff
@@ -21,6 +21,11 @@ type Connbeat struct {
 	done chan struct{}
 }
 
+type ContainerInfo struct {
+	id       string
+	localIPs mapset.Set
```

Comment:
Yes - it's a set of `string` here I think - but even if it were `net.IP` it should correctly deduplicate